lib/kubernetes-deploy/kubernetes_resource/role_binding.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class RoleBinding < KubernetesResource
+    TIMEOUT = 30.seconds
+
+    def status
+      exists? ? "Created" : "Unknown"
+    end
+
+    def deploy_succeeded?
+      exists?
```

Comment:
As far as I can tell there is no status field for this resource so the success condition seems reasonable